common/src/main/java/com/wynntils/models/wynnlanguage/WynnLanguageModel.java

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.wynnlanguage;
+
+import com.wynntils.core.components.Model;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.features.utilities.GavellianAndWynnicTransliterationFeature;
+import com.wynntils.models.activities.discoveries.DiscoveryInfo;
+import com.wynntils.models.activities.type.ActivitySortOrder;
+import com.wynntils.utils.mc.McUtils;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+import net.minecraft.ChatFormatting;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.item.ItemStack;
+
+public class WynnLanguageModel extends Model {
+    private static final int FIFTY_INDEX = 10;
+    private static final int MAX_TRANSLITERABLE_NUMBER = 5000;
+    private static final int ONE_HUNDERED_INDEX = 11;
+    private static final int TEN_INDEX = 9;
+    private static final List<Character> englishCharacters = List.of(
+            'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',
+            'v', 'w', 'x', 'y', 'z', '.', '!', '?');
+    private static final List<Integer> englishNumbers = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 50, 100);
+    private static final List<Character> gavellianCharacters = List.of(
+            'ⓐ', 'ⓑ', 'ⓒ', 'ⓓ', 'ⓔ', 'ⓕ', 'ⓖ', 'ⓗ', 'ⓘ', 'ⓙ', 'ⓚ', 'ⓛ', 'ⓜ', 'ⓝ', 'ⓞ', 'ⓟ', 'ⓠ', 'ⓡ', 'ⓢ', 'ⓣ', 'ⓤ',
+            'ⓥ', 'ⓦ', 'ⓧ', 'ⓨ', 'ⓩ');
+    private static final List<Character> wynnicCharacters = List.of(
+            '⒜', '⒝', '⒞', '⒟', '⒠', '⒡', '⒢', '⒣', '⒤', '⒥', '⒦', '⒧', '⒨', '⒩', '⒪', '⒫', '⒬', '⒭', '⒮', '⒯', '⒰',
+            '⒱', '⒲', '⒳', '⒴', '⒵', '０', '１', '２');
+    private static final List<Character> wynnicNumbers =
+            List.of('⑴', '⑵', '⑶', '⑷', '⑸', '⑹', '⑺', '⑻', '⑼', '⑽', '⑾', '⑿');
+    private static final Map<Character, Character> englishToGavellianMap = new HashMap<>();
+    private static final Map<Character, Character> englishToWynnicMap = new HashMap<>();
+    private static final Map<Character, Character> gavellianToEnglishMap = new HashMap<>();
+    private static final Map<Character, Character> wynnicToEnglishMap = new HashMap<>();
+    private static final Pattern NUMBER_PATTERN = Pattern.compile("\\d+");
+    private static final String GAVELLIAN_TRANSCRIBER_DISCOVERY = "Ne du Valeos du Ellach";
+    private static final String WYNNIC_TRANSCRIBER_DISCOVERY = "Wynn Plains Monument";
+    private static final StyledText GAVELLIAN_TRANSCRIBER = StyledText.fromString("§rHigh Gavellian Transcriber");
```

Comment:
Oh, right, Wynn already uses the term "transcription" (which is a synonym to "transliteration"). We should stick to that instead of "transliteration".

Reply:
(Sorry for requesting all these renames, but I think a good design is about 80% having a proper and well-functioning terminology)